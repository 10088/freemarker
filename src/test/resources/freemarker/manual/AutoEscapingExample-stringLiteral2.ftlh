<#-- Markup output value created by escaping plain text: -->
<#assign mo1 = "Foo & bar"?esc>
<#-- Markup output value created outherwise: -->
<#assign mo2 = "<p>Foo"?no_esc>

${"${mo1} baz"}
${"${mo2} baz"}