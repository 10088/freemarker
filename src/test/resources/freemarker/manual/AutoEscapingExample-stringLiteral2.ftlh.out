
Foo &amp; bar baz
<p>Foo baz